.circleci/config.yml

```diff
@@ -132,7 +132,6 @@ workflows:
             ]
           filters: { branches: { ignore: [main] } }
           task_name: integration_test_v2
-          tf_workspace: integration
```

Comment:
Yeah - so the job that this is part of is supposed to run against the branch env. By default if you don't pass `tf_workspace` a value it will grab the branch env as part of the default code but if you override it then it will use that as the env to run the tests against. This means since this has been here our behat code has been running against integration rather than branch envs does actually explain quite a lot of why broken code is getting to main!